### PR TITLE
Form input **PRIORITY**

### PR DESCRIPTION
**Pull request by vdvibhu20**
form-inpu1

@witty123 @abhishek97 @aayusharora

### CODE REVIEW

File: sass/styles/elements/elements.form-input.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.text-input {
+  @include dimensions(100%, 75px);
+  border: none;
+  font-size: $font-sm;
+  padding: 0 50px;
+  background-color: $nav-border;
+
+  &.icon{
+    background-image: url("/images/fingerprint.png");
```
Comment (aayusharora) [inline sass/styles/elements/elements.form-input.scss]: Please use shorthand for there properties

File: sass/styles/elements/elements.form-input.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    background-repeat: no-repeat;
+    background-size: 35px;
+    background-position-x: 30px;
+    background-position-y: center;
+    padding-left: 95px;
+  }
+}
+
+.input-btn{
```
Comment (aayusharora) [inline sass/styles/elements/elements.form-input.scss]: Not sure !! Why this is in elements rather than objects?

Comment (vdvibhu20) [reply sass/styles/elements/elements.form-input.scss]: Could have, but this is the default behaviour that is needed from text input.
So i put it in elements layer.

Comment (aayusharora) [reply sass/styles/elements/elements.form-input.scss]:
```
 &.icon{
    background-image: url("/images/fingerprint.png");
```
should be in object layer.

**Comment by vdvibhu20** [comment]
Fixed issues

@aayusharora  @witty123

**Pull request merged**